README.md

```diff
@@ -48,3 +48,15 @@ $ make
 ### Build
 
 Load `RALibretro.sln` in Visual Studio and build it.
+
+## Command Line Arguments
+
+Argument|Description
+-|-
+-c [--core]|the core's name, e.g. `--core picodrive_libretro`
+-s [--system]|the system id, see ConsoleID in [RAInterface/RA_Consoles.h](https://github.com/RetroAchievements/RAInterface/blob/master/RA_Consoles.h), e.g. `--system 1`
+-g [--game]|full path to the game's file, e.g. `--game "C:\ROMS\GEN\Demons Of Asteborg Demo.gen"`
+
+In order to successfully run a game on startup, provide the core, system and game, e.g.:
```

Comment:
I'd leave out the 'successfully'. Just because you provide all three parameters doesn't mean it will succeed. You could have a typo, or put in the wrong system id.

Reply:
✔️ https://github.com/theMK2k/RALibretro/commit/92028804fa120165b0da8bce7bc953d5c6b21204